src/make_android.sh

```diff
@@ -53,8 +53,81 @@ cordova plugin add cordova-plugin-whitelist@1.3.1
 cordova plugin add cordova-plugin-ionic-keyboard@2.2.0
 cordova plugin add https://github.com/manusimpson/Phonegap-Android-VolumeControl.git
 
+echo --- Reading arguments
+minsize=false
+full=false
+release=false
+sign=false
+os=false
+
+for i in $*; do 
+  if [ ${i%%=*} = "exclude-activities" ]
```

Comment:
Could you use TAB to indent the file, instead of SPACE

Reply:
Fixed in the latest commit